src/includes/javascript-crons-automatic-crons-instrumentation.mdx

```diff
@@ -0,0 +1,46 @@
+If you're using the [`cron`](https://www.npmjs.com/package/cron) or [`node-cron`](https://www.npmjs.com/package/node-cron) libraries to run your periodic tasks, you can use our instrumentation functions in the `Sentry.cron` export to monitor your cron jobs.
+
+### Cron
+
+Use `Sentry.cron.instrumentCron` to instrument the `CronJob` constructor or `CronJob.from` method. As a second argument to the function, you need to pass the name of the cron monitor.
+
+Requires SDK version `7.92.0` or higher.
+
+```JavaScript
+import { CronJob } from "cron";
+
+const CronJobWithCheckIn = Sentry.cron.instrumentCron(CronJob, "my-cron-job");
+
+// use the constructor
+const job = new CronJobWithCheckIn("* * * * *", () => {
+  console.log("You will see this message every minute");
+});
+
+// or from method
+const job = CronJobWithCheckIn.from({
+  cronTime: "* * * * *",
+  onTick: () => {
+    console.log("You will see this message every minute");
+  },
+});
+```
+
+### Node Cron
+
+Use `Sentry.cron.instrumentNodeCron` to instrument cron export from the `node-cron` library. This returns an object with the same API as the original `cron` export, but with the `schedule` method instrumented. You can pass the name of the cron monitor and an optional timezone as part of the third options argument to the function.
+
+Requires SDK version `7.92.0` or higher.
```

Comment:
If both options have this as a prereq, should we move this line to directly under "Automatic Crons Instrumentation"?

Reply:
Yes I'll make the change, good idea.